tests/test_user_api.py

```diff
@@ -130,7 +130,7 @@ def test_put_new_user(self):
                 }
             },
             {
-                'name': f'400 returned when creating a user that already exists',
+                'name': f'500 returned when creating a user that already exists',
```

Comment:
we should never return a 500 intentionally. What is causing this error?

Reply:
Adding a bug to track this issue

Reply:
add a link to the bug here